From 644b4f3aac650157734c66c2f4458fe963c13cb7 Mon Sep 17 00:00:00 2001
From: Andreas Mohr <and@gmx.li>
Date: Tue, 22 Dec 2020 16:09:19 +0000
Subject: [PATCH] (filegui.c) Fix -Wcast-qual warning

filegui.c:531:19: warning: cast from 'const char *' to 'char *' drops const qualifier [-Wcast-qual]
    s1 = (char *) file_date (ui->src_stat->st_mtime);
                  ^
filegui.c:546:19: warning: cast from 'const char *' to 'char *' drops const qualifier [-Wcast-qual]
    s1 = (char *) file_date (ui->dst_stat->st_mtime);
                  ^

Found by Clang-11

Signed-off-by: Andreas Mohr <and@gmx.li>
---
 src/filemanager/filegui.c | 9 +++++----
 1 file changed, 5 insertions(+), 4 deletions(-)

diff --git a/src/filemanager/filegui.c b/src/filemanager/filegui.c
index 313bb407d..de590aede 100644
--- a/src/filemanager/filegui.c
+++ b/src/filemanager/filegui.c
@@ -492,6 +492,7 @@ overwrite_query_dialog (file_op_context_t * ctx, enum OperationMode mode)
 
     vfs_path_t *p;
     char *s1;
+    const char *cs1;
     char s2[BUF_SMALL];
     int w, bw1, bw2;
     unsigned short i;
@@ -528,8 +529,8 @@ overwrite_query_dialog (file_op_context_t * ctx, enum OperationMode mode)
     size_trunc_len (s2, sizeof (s2), ui->src_stat->st_size, 0, panels_options.kilobyte_si);
     NEW_LABEL (2, s2);
     /* new file modification date & time */
-    s1 = (char *) file_date (ui->src_stat->st_mtime);
-    NEW_LABEL (3, s1);
+    cs1 = file_date (ui->src_stat->st_mtime);
+    NEW_LABEL (3, cs1);
 
     /* existing file */
     NEW_LABEL (4, dlg_widgets[4].text);
@@ -543,8 +544,8 @@ overwrite_query_dialog (file_op_context_t * ctx, enum OperationMode mode)
     size_trunc_len (s2, sizeof (s2), ui->dst_stat->st_size, 0, panels_options.kilobyte_si);
     NEW_LABEL (6, s2);
     /* existing file modification date & time */
-    s1 = (char *) file_date (ui->dst_stat->st_mtime);
-    NEW_LABEL (7, s1);
+    cs1 = file_date (ui->dst_stat->st_mtime);
+    NEW_LABEL (7, cs1);
 
     /* will "Append" and "Reget" buttons be in the dialog? */
     do_append = !S_ISDIR (ui->dst_stat->st_mode);
